bin/ged2gwb/ged2gwb.ml

```diff
@@ -713,7 +713,7 @@ EXTEND
   int:
     [ [ i = INT ->
           (try int_of_string i with Failure _ -> raise Stream.Failure)
-      | "-"; i = INT ->
```

Comment:
@hgouraud We should keep this rule and add BCE as another option

Reply:
```
  int:
    [ [ i = INT ->
          (try int_of_string i with Failure _ -> raise Stream.Failure)
      | "-"; i = INT | i = INT; ID "BCE" ->
          (try (- int_of_string i) with  Failure _ -> raise Stream.Failure) ] ]
  ;

```
would that be ok? can you do it directly on the branch